nglview/show.py

```diff
@@ -281,6 +286,29 @@ def show_schrodinger_structure(mol, **kwargs):
     return NGLWidget(structure_trajectory, **kwargs)
 
 
+def show_biopython(mol, **kwargs):
+    '''Show NGL widget with Biopython structural entity.
+
+    Takes a Structure, Model, Chain, Residue or Atom
+    from Bio.PDB as its data input.
+
+    Examples
+    --------
+    >>> import nglview as nv # doctest: +SKIP
+    ... from Bio.PDB import PDBParser
+    ... parser = PDBParser()
+    ... structure = parser.get_structure("protein", "protein.pdb")
+    ... w = nv.show_biopython(structure[0]["A"])
+    ... w
+    '''
+    io_pdb = PDBIO()
+    io_pdb.set_structure(mol)
+    io_str = StringIOOld()
+    io_pdb.save(io_str)
+    prot_str = io_str.getvalue()
+    return show_text(prot_str, **kwargs)
```

Comment:
If using that class, it's easier in the future we can add biopython's Structure directly into view

Propose idea
```
view = nv.NGLWidget()
view.add_structure(biopython_structure)
```